functions/openhabian.bash

```diff
@@ -336,3 +337,32 @@ config_ipv6() {
     if cond_redirect sysctl --load; then echo "OK"; else echo "FAILED"; return 1; fi
   fi
 }
+
+
+## Create UNIX user and group for openHABian administration purposes
+##
+##    create_user_and_group()
+##
+# IF
+# (1) the string/username that the end user entered as "username=" in openhabian.conf is *empty* OR
+# (2) the default user ("pi" on RaspiOS, "openhabian" on other OS) does not exist OR
+# (3) the user whose name the end user entered as "username=" in openhabian.conf *exists*
+#     (and isn't empty because (1) applies, too)
+# THEN skip
+# ELSE rename the default user and default group to what is defined as username= in openhabian.conf
+#
+# COMMENT:
+# according to Elias on non image installs the end user is queried to input the username:
+# https://github.com/openhab/openhabian/issues/665#issuecomment-522261443
```

Comment:
Update the comments here to remove questions that have been answered.

Reply:
ok